Global: Example addons (#1986)
pypeclub/OpenPype: openpype/modules/README.md

# OpenPype modules/addons
OpenPype modules should contain separated logic of specific kind of implementation, such as Ftrack connection and its usage code, Deadline farm rendering or may contain only special plugins. Addons work the same way currently, there is no difference between module and addon functionality.

## Modules concept
- modules and addons are dynamically imported to virtual python module `openpype_modules` from which it is possible to import them no matter where is the module located
- modules or addons should never be imported directly, even if you know possible full import path
 - it is because all of their content must be imported in specific order and should not be imported without defined functions as it may also break few implementation parts

### TODOs
- add module/addon manifest
 - definition of module (not 100% defined content e.g. minimum required OpenPype version etc.)

>  - defying that folder is content of a module or an addon

Reviewer: defying a folder as content of a module or an addon

Reviewer 2: ```suggestion
 - defining a folder as a content of a module or an addon
```
